docs/workflow.md

```diff
@@ -1,61 +1,73 @@
+- Follow the pull request template, as it helps the maintainers drive quality across the product.
+- Request at least 2 reviewers, and re-request their review when feedback has been addressed.
+- Include screenshots and PM when making UX changes.
+- Include perf traces when making performance improvements.
+- Focus on small, iterative, changes when possible.
+
+### Addressing Feedback
+- **Required**: Focus on collaboration when responding to feedback.
+- Try your best not to take feedback personally, we're all working towards the same goals.
+- When conflict arises try to address it directly with the commentor offline, preferably synchronously.
+- Provide an explanation if you decide not to implement a recommendation.
+- Use GitHub's tooling. Re-request review after feedback has been addressed.
+
+### Merging Pull Requests
+- **Required**: All comments must be resolved before merging.
+- **Required**: use the GitHub `Squash and Merge` button for this repository.
+- Merge a PR when you are confident it has been sufficiently reviewed.
+- When possible wait 24h after the last significant change before merging.
+- Ensure commit message is descriptive and helpful
 ![Good Commit Message](images/good-commit-message.png)
-
 ![Good Commit Message With More Details](images/good-commit-message-expanded.png)
-
 ![Bad Commit Message](images/bad-commit-message.png)
 
-#### Draft Pull Requests
-
-Draft pull requests are allowed, but should have a clear plan for transition to a review ready pull request.
-
-### Branching strategy
+### Providing Pull Request Feedback
+- **Required**: Focus on collaboration when providing feedback.
+- Use [suggested changes](https://docs.github.com/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request), but please never commit them for a PR you did not create.
+- Make the intent of your comment clear.
+  - ex. "Consider changing the color of this border because..."
+  - ex. "I've never seen this before! Can you help me understand..."
+  - ex. "Nit: Spelling mistake"
+- Focus on actionable feedback when requesting changes.
+  - ❌ This button won't do anything
+  - ✔️ You need to add a listener to the button for it to respond
+- Ensure the subject of your feedback is the code and not the PR author.
+- Use soft language if you must address the author directly.
+
+#### When to mark a PR as "Approved"
+- You feel confident that the code meets a high-quality bar, has adequate test coverage, and is ready to merge.
+- Any comments submitted are uncontroversial and do not need to be resolved before merging.
+- Previous feedback has been sufficiently addressed.
+#### When to leave comments without approval
+- You are not confident you can approve the PR with the requirements stated above.
+- You have feedback that you would like addressed but do not want to block the PR from merging.
+#### When to mark a PR as "Request Changes"
```

Comment:
if you edit this markdown in VS Code, and install the "markdownlint" extension, you'll get a bunch of suggestions like to add empty lines before and after any heading